docker-entrypoint.sh

```diff
@@ -27,7 +27,10 @@ envsubst <project.template.yaml> project.yaml
 
 (npm run sql || (sleep 3 && kill "$$")) &
 
-node --max-old-space-size=1536 \
+# Allow configuring node memory. Default to 1.5MB, should be ~75% of available RAM
+NODE_SPACE=${MAX_OLD_SPACE_SIZE:-1536}
```

Comment:
Shouldn't we specify this var in docker-compose so that user can understand this can be configured? Also, a line in README would be beneficial. Can you please add that?

Reply:
Maybe its better to mention in comment in the README

Reply:
I was thinking of keeping this secret since most users shouldn't need to worry about it. the default of 1536 should be good enough.

Might be good to have a section with "advanced" and put it in there